web/templates/alerts.html

```diff
@@ -9,19 +9,20 @@
     <table class="table table-striped table-bordered table-hover">
       <thead>
         <tr>
+          <th class="alert_expand_col"></th>
           <th>Alert</th>
           <th>Labels</th>
+          <th>Summary</th>
```

Comment:
I had that discussion with @grobie. He even wanted it to come first after the alertname because for him, the summary is the thing he is most interested in. Yes, it can become long, but then the table row will just become larger:

![long_summary](https://cloud.githubusercontent.com/assets/538008/9719710/57cefc8c-5587-11e5-8fa3-950e45b3b138.png)

So it can make the UI somewhat less beautiful, but wouldn't really break it.